README.md

```diff
@@ -59,3 +59,17 @@ To get started, you should take a look at the docs:
 
 https://linkedin.github.io/ConsistencyManager-iOS
 
+## Swift Version
+
+We are currently not maintaining separate branches for different Swift versions. You can use an older and stable version of the Consistency Manager for older versions of Swift though. HEAD currently supports Swift 3.
+
+| Swift Version | Consistency Manager Version  |
+|---------------|------------------------------|
+| 1             | Not supported                |
+| 2.0 - 2.1     | 2.x.x (untested)             |
+| 2.2 - 2.3     | 2.x.x                        |
+| 3 (Easy migration API) | 3.x.x               |
```

Comment:
Dropping. For such a simple table, I think it only adds work in the future to maintain.